fuzz_target_1.mayhemfile

```diff
@@ -0,0 +1,5 @@
+project: unicode-segmentation
```

Comment:
Can we rename the fuzz target to something more indicative of what's being fuzzed?